vms/administrator/models.py

```diff
@@ -24,9 +24,9 @@ class Administrator(models.Model):
             RegexValidator(r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\-)|(\.)|(,)|(\:)]+$', ),
         ],
     )
-    city = models.ForeignKey(City, null=True, blank=True)
-    state = models.ForeignKey(Region, null=True, blank=True)
-    country = models.ForeignKey(Country, null=True, blank=True)
+    city = models.ForeignKey(City, null=True, blank=True, on_delete=models.CASCADE)
+    state = models.ForeignKey(Region, null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
please remove these changes

Reply:
On upgrading django to 2.0 it is necessary to add on_delete method. For django 1.1 default is cascade therefore I add on_delete method to model cascade.
You can check out in django documentation also.

Reply:
@yashshah2820